### PR TITLE
chore: repository standardization

### PR DESCRIPTION
**Pull request by bloodf**
### Short Description of the PR
this PR will introduce the documentation for the new open-source standards in our public repositories and add the missing docs in our repository.

removed shopify from the core docs, as they now have their own docs deployment and url

### Pull Request Checklist

- [ ] I have updated the Changelog ([V1](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md)) [v2](https://docs-next.vuestorefront.io/contributing/creating-changelog.html) and mentioned all breaking changes in the public API.
- [ ] I have documented all new public APIs and made changes to existing docs mentioning the parts I've changed so they're up to date.
- [ ] I have tested my Pull Request on production build and (to my knowledge) it works without any issues

- I tested manually my code and it works well with both:
- [ ] Default Theme
- [ ] Capybara Theme
- [ ] I have written test cases for my code

- [x] I have followed [naming conventions](https://github.com/kettanaito/naming-cheatsheet)

### CODE REVIEW

**Review by filrak — APPROVED** [review]

File: packages/core/docs/integrate/boilerplate.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Using the integration boilerplate
```
Comment (filipsobol) [inline packages/core/docs/integrate/boilerplate.md:1]: This duplicates our [Integration Guide](https://docs.vuestorefront.io/v2/integrate/integration-guide.html)

File: packages/core/docs/integrate/cache-driver.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Integrating cache driver
 
 ::: warning Want to build an integration?
-If you want to integrate with Vue Storefront, please **contact the core team on our [slack](https://slack.vuestorefront.io)** first. We are eager to help you with building it and ensuring its high quality! Building the integration together with the core team is the best way to keep its quality high and make it officially recommended once it's done.
+If you want to integrate with Vue Storefront, please **contact the core team on our [discord](https://discord.vuestorefront.io/)** first. We are eager to help you with building it and ensuring its high quality! Building the integration together with the core team is the best way to keep its quality high and make it officially recommended once it's done.
```
Comment (filipsobol) [inline packages/core/docs/integrate/cache-driver.md:4]:
```suggestion
If you want to integrate with Vue Storefront, please **contact the core team on our [Discord](https://discord.vuestorefront.io/)** first. We are eager to help you with building it and ensuring its high quality! Building the integration together with the core team is the best way to keep its quality high and make it officially recommended once it's done.
```

File: packages/core/docs/integrate/cms.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Integrating CMS
 
 ::: warning Want to build an integration?
-If you want to integrate with Vue Storefront, please **contact the core team on our [slack](https://slack.vuestorefront.io)** first. We are eager to help you with building it and ensuring its high quality! Building the integration together with the core team is the best way to keep its quality high and make it officially recommended once it's done.
+If you want to integrate with Vue Storefront, please **contact the core team on our [discord](https://discord.vuestorefront.io/)** first. We are eager to help you with building it and ensuring its high quality! Building the integration together with the core team is the best way to keep its quality high and make it officially recommended once it's done.
```
Comment (filipsobol) [inline packages/core/docs/integrate/cms.md:4]:
```suggestion
If you want to integrate with Vue Storefront, please **contact the core team on our [Discord](https://discord.vuestorefront.io/)** first. We are eager to help you with building it and ensuring its high quality! Building the integration together with the core team is the best way to keep its quality high and make it officially recommended once it's done.
```

File: packages/core/docs/integrate/integration-guide.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # Integration guide
 
 ::: warning Want to build an integration?
-If you want to integrate with Vue Storefront, please **contact the core team on our [slack](https://slack.vuestorefront.io)** first. We are eager to help you with building it and ensuring its high quality! Building the integration together with the core team is the best way to keep its quality high and make it officially recommended once it's done.
+If you want to integrate with Vue Storefront, please **contact the core team on our [discord](https://discord.vuestorefront.io/)** first. We are eager to help you with building it and ensuring its high quality! Building the integration together with the core team is the best way to keep its quality high and make it officially recommended once it's done.
```
Comment (filipsobol) [inline packages/core/docs/integrate/integration-guide.md:4]:
```suggestion
If you want to integrate with Vue Storefront, please **contact the core team on our [Discord](https://discord.vuestorefront.io/)** first. We are eager to help you with building it and ensuring its high quality! Building the integration together with the core team is the best way to keep its quality high and make it officially recommended once it's done.
```

**Review by filipsobol — CHANGES_REQUESTED** [review]

**Review by andrzejewsky — APPROVED** [review]

File: packages/core/docs/integrate/boilerplate.md (hunk 1 above)
Comment (bloodf) [reply packages/core/docs/integrate/boilerplate.md:1]: It was removed from that part to the new one.

**Review by filipsobol — APPROVED** [review]

**Pull request merged**